test/screenshot/infra/types/node-externs.js

```diff
@@ -81,8 +81,8 @@
 
 /**
  * @typedef {{
- *   pid: number,
- *   ppid: number,
+ *   pid: string,
```

Comment:
FYI: The `ps-node` API actually returns a `string` value for both `pid` and `ppid`.

I was previously calling `parseInt()` and overwriting the `string` values with `number`, but that's now removed.